<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteEditorWindow</class>
 <widget class="QMainWindow" name="SpriteEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>875</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SpriteEditorWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="chooseColorBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>435</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Color</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>425</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px rgb(238, 238, 236);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="resolutionSlider">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="sliderPosition">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="resolutionLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>490</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="frameLayout"/>
   </widget>
   <widget class="QListWidget" name="framesList">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>580</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::CopyAction</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>780</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFrameButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>780</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="penButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>340</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="eraserButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="selectionButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>780</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
   </widget>
   <widget class="QLabel" name="framesListLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frames</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="drawMirrorCheckBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>380</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Mirrored</string>
    </property>
   </widget>
   <widget class="QSlider" name="frameRateSlider">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="frameRateLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame Rate (fps)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="popOutButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>250</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Size Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="penIcon">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/penIcon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="eraserIcon">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>380</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/eraserIcon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="selectionIcon">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>340</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/selectionIcon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="mirrorIcon">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>380</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/mirrorIcon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="itemUpButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>620</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string extracomment="Dave Gandy - flaticon.com">Dave Gandy - flaticon.com</string>
    </property>
    <property name="toolTipDuration">
     <number>10000</number>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/upIcon.png</normaloff>:/images/upIcon.png</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="itemDownButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>710</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string notr="true">Dave Gandy - flaticon.com</string>
    </property>
    <property name="toolTipDuration">
     <number>10000</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string extracomment="Icon made by Dave Gandy of flaticon.com"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/downIcon.png</normaloff>:/images/downIcon.png</iconset>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1095</x>
      <y>660</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move Item</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="previewLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
